Source/ThirdParty/Autolinker.js

```diff
@@ -527,9 +527,9 @@ Autolinker.Util = {
 	/**
 	 * Assigns (shallow copies) the properties of `src` onto `dest`.
 	 *
-	 * @param {Object} dest The destination object.
```

Comment:
Revert changes to this file.  We don't need to worry about how third party libraries do their documentation